### DISCUSSION TITLE
How to estimate risk of a dental cleaning and checkup visit?

### DISCUSSION THREAD

**Discussion by 366anna** [microCOVID/microCOVID#1129] [Q&A] [answered]
Dentist is very close to your face, and they spend all day at this distance from many unmasked people.
Would they be the same risk as an average person in the area, higher, or lower? Or maybe it would be closer to someone who went to a bar?

**Comment by blanchardjeremy** [comment, marked as answer]
We've been counting medical professionals as "average" -- the few studies we've had a chance to look at show that they can actually be lower risk. Though I don't know if we've seen any specific to dentists. 

They're usually taking more precautions than an average person. Like HEPA filters in the office and such.
